### ISSUE TITLE
Annotation of MPI-INF-3DHP and VNect output doesn't match

### ISSUE THREAD

**Issue by aingo03304** [XinArkh/VNect#14]
First of all, thank you for your work on this repo. It is really helpful.

Recently, I am implementing VNect paper on tensorflow v2, but struggling for preprocessing the data.
I found that the scale of MPI-INF-3DHP (approx. -1000 ~ 1000 on rel. position) and VNect output (approx. -5 ~ 5 on rel. pos.) doesn't match correctly.
Can I get some information about this scaling process?

Thanks in advance!

**Comment by XinArkh** [comment]
Can you explain more about the scale you mentioned? I cannot understand it well. Thanks!

**Comment by aingo03304** [comment]
@XinArkh Thank you for the quick response!
WIth MPI-INF-3DHP, you can check the range for the relative position value of 3d annotation is like -1000~1000.
But, passing inference of VNect, you can check the range of the output is like -5~5.

**Comment by aingo03304** [comment]
@XinArkh I have preprocessed like this.
1. Get annot3 from annot.mat
2. Subtract annot3 by the pelvis location values.
Then, you can get values between -1000 and 1000.
However, these values are too big to train, and VNect also doesn't fit these values' range.
So, there must be a certain scaling method for handling this issue but I cannot find any information related to this issue.

**Comment by XinArkh** [comment]
VNect output is supposed to be in decimeter unit according to the following code:
https://github.com/XinArkh/VNect/blob/8fef417edd02f783e78aaae2edcadc2d6addbceb/src/utils.py#L192

In MPI-INF-3DHP, univ_annot3 instead of annot3 is recommended. univ_annot3 normalized the length from knee to neck along the kinematic chain to a fixed 92cm, according to the paper author. Given this fact, you can have a look to find its length scale.

For more questions, you are welcome to give more comments here.

**Comment by aingo03304** [comment]
@XinArkh Thank you for the kind reply!
